pywb/rewrite/rewriteinputreq.py

```diff
@@ -93,11 +93,13 @@ def get_req_headers(self):
                 if self.splits:
                     value = self.splits.scheme
 
-            elif not has_brotli and name == 'HTTP_ACCEPT_ENCODING' and 'br' in value:
+            elif name == 'HTTP_ACCEPT_ENCODING':
                 # if brotli not available, remove 'br' from accept-encoding to avoid
                 # capture brotli encoded content
+                # We have to remove zstd from the list of accepted encodings as warcio does not support it.
+                disallowed_encodings = ('zstd') if has_brotli else ('zstd', 'br')
```

Comment:
I think this needs a comma:

```suggestion
                disallowed_encodings = ('zstd',) if has_brotli else ('zstd', 'br')
```

Otherwise if `has_brotli` is true, then `disallowed_encodings` is the string `"zstd"` not a tuple which means the `not in disallowed_encodings` is checking for any substring of 'zstd'.

```
>>> 'z' not in ('zstd')
False
>>> 'z' not in ('zstd',)
True
```

Reply:
Good catch